client/src/App.js

```diff
@@ -1,8 +1,11 @@
 import React, { useState } from 'react';
+import {Route} from 'react-router-dom';
+import MovieList from './Movies/MovieList';
+import Movie from './Movies/Movie';
 
 import SavedList from './Movies/SavedList';
 
-const App = () => {
+const App = (props) => {
```

Comment:
props only needs to be passed into a component when that component is receiving props. Since App is the parent component of all the other components, it is not receiving props from any component.